src/styles/blocks/product-card.css

```diff
@@ -0,0 +1,102 @@
+.imac--position {
+  display: flex;
+  flex-direction: column;
+}
+
+.imac {
+  min-height: 408px;
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-image--position {
+  margin: 32px 19px 40px;
+}
+
+.product-image {
+  width: 160px;
+  height: 134px;
+}
+
+.full-name--position {
+  margin: 0 auto 23px;
+}
+
+.full-name {
+  width: 166px;
+  height: 36px;
+  line-height: 18px;
+
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.product-code--position {
+  margin: 0 90px 18px 16px;
+}
```

Comment:
Bad idea to use margins to stick your element to the left side, find out another solution